crypto/asn1/a_int.c

```diff
@@ -289,7 +289,7 @@ static int asn1_get_int64(int64_t *pr, const unsigned char *b, size_t blen,
             ASN1err(ASN1_F_ASN1_GET_INT64, ASN1_R_TOO_SMALL);
             return 0;
         }
-        *pr = -(int64_t)r;
+        *pr = -(uint64_t)r;
```

Comment:
Right, it was VC... As for it being done in other places. Well, the only legitimate reason to disregard warning is that compiler gets it wrong, i.e. there is bug in warning subsystem. This warning doesn't appear as bug in compiler warning subsystem, and hence we should be better off if warnings in question are addressed, rather than ignored. It doesn't have to be here. And if pull request doesn't show up shortly I'll create one...